### PR TITLE
Issue #SB-24225 feat: Ability for group admin to view activity dashboard

### PR DESCRIPTION
**Pull request by kamaleeswary**

### CODE REVIEW

File: src/app/client/package.json [hunk 1]
```diff
@@ -118,6 +118,7 @@
     "ngx-order-pipe": "^2.0.4",
     "ngx-slick": "0.2.1",
     "rxjs": "^6.5.5",
+    "sb-dashlets": "^1.0.3",
```
Comment (balakrishna10) [inline src/app/client/package.json:121]: "@project-sunbird/sb-dashlet": "^1.0.0"

Comment (kamaleeswary) [reply src/app/client/package.json:121]: ok

File: src/app/client/angular.json [hunk 2]
```diff
@@ -79,6 +79,7 @@
               "node_modules/jquery/dist/jquery.min.js",
               "node_modules/slick-carousel/slick/slick.min.js",
               "node_modules/@project-sunbird/telemetry-sdk/index.js",
+              "node_modules/datatables.net/js/jquery.dataTables.js",
```
[the remark this answers is not in the document]
Comment (kamaleeswary) [reply src/app/client/angular.json:82]: it is dashlet dependency

File: src/app/client/src/app/modules/dashboard/components/data-table/data-table.component.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { IColDefination, IDataTableOptions } from './../../interfaces';
 import { Component, Input, AfterViewInit, Output, EventEmitter } from '@angular/core';
 import * as $ from 'jquery';
-import * as datatable from 'datatables.net';
+import 'datatables.net';
```
[the remark this answers is not in the document]
Comment (kamaleeswary) [reply src/app/client/src/app/modules/dashboard/components/data-table/data-table.component.ts:4]: yes, we got an error here, after installing sb-dashlets, so Ravinder suggests to use this.

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**